LEAF_Request_Portal/utils/LEAF_importStandardConfig.php

```diff
@@ -100,6 +102,11 @@ function importTable($db, $tempFolder, $table) {
 importTable($db, $tempFolder, 'workflow_routes');
 importTable($db, $tempFolder, 'step_modules');
 
+// remove files from templates/email/custom_override on subordinate
+passthru('rm -rf ' . $paste_custom_templates . '*');

```

Comment:
1. There should be a check to ensure the folder exists.
2. PHP Filesystem Functions should be used instead of using OS-specific commands.

Reply:
Done.